packages/sado-connect/src/components/SadoConnectKit.tsx

```diff
@@ -31,7 +31,7 @@ export function SadoConnectKit({
         <PreConnectButton openModal={openModal} customStyle={customStyle} />
       ) : (
         <PostConnectButton
-          address={address}
+          address={address.ordinals}
```

Comment:
do we need to check that `address.ordinals` exists as well?

Reply:
fixed